packages/widgets/src/components/Pagination/styles.tsx

```diff
@@ -0,0 +1,81 @@
+import { css, Theme } from '@emotion/react';
+
+const baseBtnStyles = (theme: Theme) => css`
+  height: 25px;
+  border-radius: 6px;
+
+  ${theme.mediaQueries.xs} {
+    height: 30px;
+  }
+`;
+
+export const pageBtnStyles = (theme: Theme) => css`
+  ${baseBtnStyles(theme)}
+
+  background: unset;
+  box-shadow: unset;
+  color: #070821;
+  padding: 0 9px;
+
+  ${theme.mediaQueries.xs} {
+    padding: 0 11px;
+  }
+
+  &:hover,
+  &:active,
+  &:focus {
+    background: #eef1f7;
+    box-shadow: unset;
+  }
+`;
+
+const selectedBtnBg = (theme: Theme) => css`
+  background: ${theme.colors.blueLightDarker};
+  background: linear-gradient(
+    247.37deg,
+    ${theme.colors.blueDark},
+    ${theme.colors.blueLightDarker}
+  );
+`;
+
+export const selectedPageBtnStyles = (theme: Theme) => css`
+  ${baseBtnStyles(theme)}
+
+  ${selectedBtnBg(theme)}
+
+  color: white;
+  margin: 0 3px;
+  padding: 0 10px;
+
+  ${theme.mediaQueries.xs} {
+    padding: 0 13px;
+  }
+
+  &:hover {
+    box-shadow: 0 5px 5px -1px rgba(0, 0, 0, 0.3);
+    cursor: default;
+  }
+
+  &:hover,
+  &:active,
+  &:focus {
+    ${selectedBtnBg(theme)}
+  }
+`;
+
+export const arrowBtnStyles = (theme: Theme) => css`
+  ${baseBtnStyles(theme)}
+
+  padding: 0 8px;
+  background: #eef1f7;
```

Comment:
Could we use color from theme.colors?

Reply:
ok, will do. Initially, I wanted to avoid adding such colors to the theme. But I think the addition is worth doing: it will be easier to refactor when we come to implementing the full-fledge theming approach we discussed.

Reply:
It makes sense. In this case, we can keep in place just color, instead of using `theme.colors.`